app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -29,11 +29,13 @@ def save_metadata(metadata_hash)
   end
 
   def rrd_status
-    return :processed if rrd_claim_processed?
+    return 'processed' if rrd_claim_processed?
 
-    return :pending_ep if form.dig('rrd_metadata', 'offramp_reason') == 'pending_ep'
+    return form.dig('rrd_metadata', 'offramp_reason') if form.dig('rrd_metadata', 'offramp_reason').present?
```

Comment:
Here is where we return a string value, so let's return strings for the other return possibilities.